### PR TITLE
Re-design plataform 

### PR DESCRIPTION
**Pull request by vitormancio**

### CODE REVIEW

File: pages/auth/index.js [hunk 1]
```diff
@@ -242,23 +239,30 @@ function authPage() {
...
-                id={'sign-in-with-google'}
-                imgSrc={'/assets/img/google-logo.svg'}
-                alt="Google-Login-Icon"
-                loginGoogle={() => loginGoogle()}
-                textContent={'Login com o Google'}
-                imgSize={'40'}
-              />
-            </div>
+                <div className='flex flex-col gap-3'>
+                  <Button
```
Comment (danicuki) [inline pages/auth/index.js:243]: perdeu o id do botão aqui

File: pages/auth/signup/index.js [hunk 2]
```diff
@@ -160,22 +163,29 @@ function signUpPage() {
...
-              <LoginButton
-                id={'sign-in-with-google'}
-                imgSrc={'/assets/img/google-logo.svg'}
-                alt="Google-Login-Icon"
-                textContent={'Login com o Google'}
-                loginGoogle={() => loginGoogle()}
-                imgSize={'40'}
-              />
+              <div className='flex flex-col gap-3'>
+                  <Button
```
Comment (danicuki) [inline pages/auth/signup/index.js:167]: aqui também perdeu o id

File: pages/courses/index.js [hunk 3]
```diff
@@ -1,33 +1,81 @@
...
-                id={c?.id}
-                img={c?.image_url}
-                title={c?.title}
-                desc={c?.description}
-                difficulty={c?.difficulty}
-                duration={c?.duration}
-                tags={c.tags}
-              />
+              <>
+                <Card
```
Comment (danicuki) [inline pages/courses/index.js:31]: aqui também perdeu o id

File: components/Navbar/index.js [hunk 4]
```diff
@@ -32,50 +39,51 @@ export default function Navbar() {
...
+                <Image width={42} height={42} src="/assets/img/w3d-logo-symbol-ac.svg" />
+                <Text weight={'bold'}>WEB3DEV</Text>
+                <ThemeSwitch />
+              </Navbar.Brand>
+            </Link>
 
-            {process.env.NEXT_PUBLIC_ENVIRONMENT && (
-              <div className="w-1/3 bg-gradient-to-r from-green-400 to-violet-500 text-center font-semibold tracking-widest">
+            {/* <Navbar.Content hideIn={'md'} >
+              {process.env.NEXT_PUBLIC_ENVIRONMENT && (
```
Comment (Lorenzobattistela) [inline components/Navbar/index.js:60]: we need to avoie commented code along the code, are we going to use this?

**Review by Lorenzobattistela — COMMENTED** [review]
started revision

File: components/Navbar/index.js (hunk 4 above)
Comment (vitormancio) [reply components/Navbar/index.js:60]: This is one development state refear on Navbar, and make sense to maintain this code ! Thanks 😄

File: components/Navbar/index.js [hunk 5]
```diff
@@ -1,24 +1,31 @@
...
+import { 
+  FaEthereum
+ } from 'react-icons/fa'
 
 import { useSession, signOut } from 'next-auth/react'
 import { getUserFromFirestore, updateUserInFirestore } from '../../lib/user'
 import { checkReferral, saveReferralCookie } from '../../lib/store_referral'
 
-export default function Navbar() {
+export default function NavbarComponent() {
```
[the remark this answers is not in the document]
Comment (vitormancio) [reply components/Navbar/index.js:18]: That's because the sample component of Navbar from NextUI have name : `Navbar` and its the same ...

**Comment by vitormancio** [comment]
@Lorenzobattistela It's important run locally this branch to help help see probably problems 😄

File: components/Navbar/index.js (hunk 4 above)
Comment (Lorenzobattistela) [reply components/Navbar/index.js:60]: Can you explain when/why are we using it? just curious 😄

File: components/ThemeSwitch/index.js [hunk 6]
```diff
@@ -1,36 +1,30 @@
-import { useTheme } from 'next-themes'
 import { useState, useEffect } from 'react'
 import { MoonIcon, SunIcon } from '@heroicons/react/solid'
+import { useTheme as useNextTheme } from 'next-themes'
+import { Switch, useTheme } from '@nextui-org/react'
+
```
Comment (Lorenzobattistela) [inline components/ThemeSwitch/index.js:5]: [style] empty line

File: components/ThemeSwitch/index.js [hunk 7]
```diff
@@ -1,36 +1,30 @@
...
-          className="h-5 w-5 text-primary-300"
-        />
-      )}
-    </button>
-  )
+    const { setTheme } = useNextTheme();
+    const { isDark, type } = useTheme();
+
+  return <Switch 
+  shadow
```
Comment (Lorenzobattistela) [inline components/ThemeSwitch/index.js:22]: lost datatest Id

File: pages/_app.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import '../styles/globals.css'
 import Head from 'next/head'
-import { ThemeProvider } from 'next-themes'
+import { ThemeProvider as NextThemesProvider } from 'next-themes';
```
Comment (Lorenzobattistela) [inline pages/_app.js:3]: [style] `;` char

File: pages/_document.js [hunk 9]
```diff
@@ -1,17 +1,21 @@
 import Document, { Html, Head, Main, NextScript } from 'next/document'
+import { CssBaseline } from '@nextui-org/react';
```
Comment (Lorenzobattistela) [inline pages/_document.js:2]: [style] `;` char

File: pages/auth/index.js [hunk 10]
```diff
@@ -56,14 +59,8 @@ function authPage() {
       <Head>
         <title>Login -Web3Dev</title>
       </Head>
-      <div className="bg-gray-50 dark:bg-black-300">
+      <div className="">
```
Comment (Lorenzobattistela) [inline pages/auth/index.js:62]: [style] empty classname

File: pages/auth/index.js [hunk 11]
```diff
@@ -77,10 +74,10 @@ function authPage() {
...
             </>
           ) : (
-            <div className="w-full rounded bg-white-100 px-6 py-6 shadow-lg dark:bg-black-200 sm:px-6 sm:py-10 md:w-1/2 lg:w-5/12 lg:px-10 xl:w-1/3">
+            <div className="w-full rounded  px-6 py-6 shadow-lg m:px-6 sm:py-10 md:w-1/2 lg:w-5/12 lg:px-10 xl:w-1/3">
               <p
                 tabIndex={0}
-                className="text-2xl font-extrabold leading-6 focus:outline-none dark:text-white-100"
+                className="text-2xl font-extrabold leading-6 focus:outline-none"
               >
                 {isDisable ? 'Resete sua senha' : 'Entre na sua conta'}
```
Comment (Lorenzobattistela) [inline pages/auth/index.js:82]: maybe Recuperar senha fits better? What do you think?

File: pages/auth/signup/index.js [hunk 12]
```diff
@@ -6,7 +6,10 @@ import { useForm } from 'react-hook-form'
 import { toast } from 'react-toastify'
 import useAuth from '../../../hooks/useAuth'
 import { withPublic } from '../../../hooks/route'
-import LoginButton from '../../../components/LoginButton'
+import { Button } from '@nextui-org/react'
+
```
Comment (Lorenzobattistela) [inline pages/auth/signup/index.js:10]: [style] empty line

File: pages/auth/signup/index.js [hunk 13]
```diff
@@ -31,24 +34,24 @@ function signUpPage() {
         <Head>
           <title>Cadastro -Web3Dev</title>
         </Head>
-        <div className="bg-gray-50 dark:bg-black-300">
+        <div className="">
```
Comment (Lorenzobattistela) [inline pages/auth/signup/index.js:37]: [style] empty classname

**Comment by Lorenzobattistela** [comment]
Are you developing based on a forked repository on your local machine?

File: components/ThemeSwitch/index.js (hunk 7 above)
Comment (vitormancio) [reply components/ThemeSwitch/index.js:21]: I change the icon libary and need recreate this datatest and test

**Comment by vitormancio** [comment]
@Lorenzobattistela I'm ajust all recommended reviews 😄

**Review by Lorenzobattistela — APPROVED** [review]

**Comment by vitormancio** [comment]
Thanks @Lorenzobattistela , i need ajust one final thing on referal link and i submit de Pull Request

**Comment by vitormancio** [comment]
@Lorenzobattistela I'm ajust on globalcss for all buttons have `cursor:pointer` thanks for feedback !

**Pull request merged**